views/create_peep.erb

```diff
@@ -0,0 +1,9 @@
+<div class="centered-content">
+  <form action="/new-peep" method="post">
+    <h2 for="message" class="create-peep-header">Message:</h2>
+    <input type="text" name="message" id="message" placeholder="What's on your mind?" maxlength="60"required>
```

Comment:
Does maxlength say that the user can only put in 60 characters? If so, that's really smart